Command/LoadFixtureCommand.php

```diff
@@ -136,7 +135,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ($dirOrFile) {
             $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
         } else {
-            /** @var $kernel KernelInterface */
+            /** @var $kernel \Symfony\Component\HttpKernel\KernelInterface */
```

Comment:
lets keep the use statement for this. we are using the kernel in this class so the use statement gives a hint about that.